### DISCUSSION TITLE
New Units

### DISCUSSION THREAD

**Discussion by im-mortal** [Warhammer-Mods/arkhan_stevie#11] [Ideas]
Please use this to discuss and propose new units to add.

---
As per Stephen's initial suggestion:

- [ ] Nehekharan Warriors (Shields)
- [ ] Asaph Archers
- [ ] Necropolis Mounts for Necrotects (Khalida's Faction)

**Comment by im-mortal** [comment]
Zombie Dragon: #12

**Comment by Zalbardeon** [comment]
When it comes to the unit card for the Asaph Archers, I was thinking of using the base Skeleton Archer unit card, and taking the helmet from the Tomb Guard, with a slightly different sandy background. 

Nehekharan Warriors (Shields) could take a shield icon from the Skeleton Spearmen or Tomb Guard.
